src/test/java/io/vertx/test/core/BufferTest.java

```diff
@@ -605,6 +605,7 @@ public void testGetBytes() throws Exception {
     Buffer b = Buffer.buffer(bytes);
 
     assertTrue(TestUtils.byteArraysEqual(bytes, b.getBytes()));
+    assertEquals(100, b.getBytes(0, 99).length);
```

Comment:
The point I was making was the API docs for the method is a bit vague. The first parameter being inclusive and the second exclusive.